Cubical/HITs/FiniteMultiset/Order.agda

```diff
@@ -0,0 +1,187 @@
+{-# OPTIONS --cubical --safe #-}
+module Cubical.HITs.FiniteMultiset.Order where
+
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+open import Cubical.Data.Nat
+open import Cubical.Data.Nat.Order
+open import Cubical.Data.Empty as ⊥
+open import Cubical.Data.Sum
+open import Cubical.Relation.Nullary
+open import Cubical.HITs.FiniteMultiset.Base
+open import Cubical.HITs.FiniteMultiset.Properties as FMS
+open import Cubical.Structures.MultiSet
+open import Cubical.Relation.Nullary.DecidableEq
+
+
+private
+ variable
+  A : Type₀
```

Comment:
I don't see a problem with that, it might just have been lazy since in the original definition of FMSet it requires A : Type₀ but I don't see why this can't be generalized.

Reply:
Ah ok, can you please generalize all of the different versions?